docs/operator-guide/microsoft-entra/oidc-authentication-overview.md

```diff
@@ -12,7 +12,7 @@ Below is the list of components with OIDC integration instructions:
 
 - [Ansible AWX](awx-operator-authentication.md)
 - [Argo CD](argo-cd-authentication.md)
-- [AWS EKS and KubeRocketCI Portal](/blog/integrating-oidc-authentication-microsoft-entra-aws-eks)
+- [AWS EKS and KubeRocketCI Portal](aws-eks-portal-authentication.md)
```

Comment:
please do not change the link name, this will have negative impact on google search

Reply:
please ignore, everything is ok